--- origsrc/libxaw3dxft-1.3.3/Makefile.am	2012-02-05 09:38:19.000000000 -0600
+++ src/libxaw3dxft-1.3.3/Makefile.am	2012-02-27 22:07:01.093253500 -0600
@@ -59,7 +59,7 @@ laylex.c: laylex.l
 # referenced in the usual roundabout way, but that broke make dist.
 symlink.bs:
 	$(MKDIR_P) X11
-	(cd X11 && $(LN_S) "../$(srcdir)" Xaw3dxft)
+	(cd X11 && $(LN_S) "$(top_srcdir)" Xaw3dxft)
 	touch symlink.bs
 
 clean-local:
@@ -68,7 +68,7 @@ clean-local:
 
 install-exec-hook:
 	-mkdir -p $(DESTDIR)/usr/lib/pkgconfig
-	cat libxaw3dxft.pc.in | sed -e "/VERSION/s//$(VERSION)/" > libxaw3dxft.pc
+	cat $(srcdir)/libxaw3dxft.pc.in | sed -e "/VERSION/s//$(VERSION)/" > libxaw3dxft.pc
 	/bin/cp -p libxaw3dxft.pc $(DESTDIR)/usr/lib/pkgconfig
 	rm -f libxaw3dxft.pc
 
